README.md

```diff
@@ -6,10 +6,90 @@ Tools for coordinating independent single-page-apps embedded via iframe
 
 Iframes are the only way to obtain strong isolation of the javascript runtime in a browser. This is useful for organizations with multiple teams shipping interfaces for different features. Teams can ship with confidence, knowing errors in other's code won't crash their interface.
 
-Embedding applications via iframe also means that code from separate teams can be deployed and rolled back independently, limiting the impact of breaking changes. 
-
+Embedding applications via iframe also means that code from separate teams can be deployed and rolled back independently, limiting the impact of breaking changes.
 
 ## How do I use it?
```

Comment:
I think ultimately we'll want a brief intro on the concepts and architecture here to prime the reader for usage instructions.  Eventually, this bried description could also link out to architecture and API documents.